.expeditor/buildkite/kubernetes.sh

```diff
@@ -74,3 +74,22 @@ if [[ `grep -c "^kind: Service$" kubernetes/deployment.yml` -gt 0 ]]; then
   echo "+++ Waiting for Load Balancer..."
   wait_for_elb
 fi
+
+# This block tests the "environment" urls
+if [ "$ENVIRONMENT" == "acceptance" ]; then
+  curl --fail -I "https://omnitruck-acceptance.chef.io/_status"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/install.sh"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/install.ps1"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/versions"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/metadata?p=ubuntu&pv=18.04&m=x86_64"
+  curl --fail -I "https://omnitruck-acceptance.chef.io/stable/chef/download?p=ubuntu&pv=18.04&m=x86_64"
+elif [ "$ENVIRONMENT" == "production" ]; then
+  curl --fail -I "https://omnitruck.chef.io/_status"
+  curl --fail -I "https://omnitruck.chef.io/install.sh"
+  curl --fail -I "https://omnitruck.chef.io/install.ps1"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/versions"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/metadata?p=ubuntu&pv=18.04&m=x86_64"
+  curl --fail -I "https://omnitruck.chef.io/stable/chef/download?p=ubuntu&pv=18.04&m=x86_64"
+else
+  echo "No URL verification done for $ENVIRONMENT"
+fi
```

Comment:
Placing the smoke checks in this file is redundant....we execute `.expeditor/buildkite/URL-verification.sh` directly after the deployment to k8s.